code/modules/uplink/uplink_item.dm

```diff
@@ -1319,6 +1319,13 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 	desc = "Strong flavor, dense smoke, infused with omnizine."
 	item = /obj/item/weapon/storage/fancy/cigarettes/cigpack_syndicate
 	cost = 2
+	
+/datum/uplink_item/badass/syndiespinner
+	name = "Syndicate Fidget Spinner"
+	desc = "This fidget spinner is designed to be a little more robust than others like it. \
+			Its bearings are much smoother to allow for faster spinning, and its blades are actual blades."
```

Comment:
"Its blades are actually blades"
Could be reworded, "its blades are actually sharp" would be better imo